trunk/admin.widget.class.php

```diff
@@ -153,6 +154,10 @@ public function sanitize($input)
             $new_input['from_name'] = trim($input['from_name']);
         }
 
+        if (!empty($input['template'])) {
+            $new_input['template'] = trim($input['template']);
```

Comment:
Not sure I understand why we'd need `esc_attr` - it escapes input, specifically "less than, greater than, ampersand, double quote and single quote". I don't anticipate any of those in this field. Maybe I'm just missing something about this function?